ftrack_s3_accessor/s3.py

```diff
@@ -12,6 +12,7 @@
     pass
 
 from ftrack_api.accessor.base import Accessor
+from ftrack_api.exception import AccessorFilesystemPathError
```

Comment:
This can be merged with the other imports from ftrack_api.exception

Reply:
done !